src/components/shared/FormDialog.tsx

```diff
@@ -6,30 +6,41 @@ import { useForm, SubmitHandler, UseFormReturn, FieldValues } from 'react-hook-f
 import { yupResolver } from '@hookform/resolvers/yup'
 import * as React from 'react'
 import * as yup from 'yup'
+import HorizontalStepper from '../features/Stepper'
+import { useIsSmallScreen } from '@/hooks/useIsSmallScreen'
+import { useTranslation } from 'react-i18next'
 
 interface FormDialogProps<T extends FieldValues> {
   open: boolean
   title: string
+  steps?: string[]
+  activeStep?: number
```

Comment:
you are overcomplicating the FormDialog component with stepper, which it shouldnt handle such cases. Try to think of something else and use this at an upper-level component.

Reply:
Fixed